### PR TITLE
Fix misspelled codgen package in tests

### PR DESCRIPTION
**Pull request by ZacSweers**
This has always bothered me

### CODE REVIEW

File: kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt [hunk 1]
```diff
@@ -27,6 +27,7 @@ import com.squareup.moshi.Moshi
 import com.squareup.moshi.ToJson
 import com.squareup.moshi.Types
 import com.squareup.moshi.internal.NullSafeJsonAdapter
+import com.squareup.moshi.kotlin.codgen.GeneratedAdaptersTest_MultiplePropertiesShareAdapterJsonAdapter
```
Comment (rharter) [inline kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt]: Is this right or an auto import from before a recompile?

Comment (ZacSweers) [reply kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt]: grrr autoimport from IDE

Comment (ZacSweers) [reply kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt]: 795b08b

File: kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt [hunk 2]
```diff
@@ -101,7 +103,8 @@ class GeneratedAdaptersTest {
     }
 
     @Language("JSON") val expected = """{"foo":"fooString","bar":"","bazList":[]}"""
-    assertThat(adapter.toJson(DefaultValues("fooString"))).isEqualTo(expected)
+    assertThat(adapter.toJson(
+        DefaultValues("fooString"))).isEqualTo(expected)
```
Comment (swankjesse) [inline kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt]: what’s with the weird wrapping?

File: kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt [hunk 3]
```diff
@@ -340,7 +345,8 @@ class GeneratedAdaptersTest {
     val moshi = Moshi.Builder().build()
     val jsonAdapter = moshi.adapter<ConstructorParametersAndProperties>()
 
-    val encoded = ConstructorParametersAndProperties(3)
+    val encoded = ConstructorParametersAndProperties(
+        3)
```
Comment (swankjesse) [inline kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt]: this seems like madness

File: kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt (hunk 2 above)
Comment (ZacSweers) [reply kotlin/tests/src/test/kotlin/com/squareup/moshi/kotlin/codegen/GeneratedAdaptersTest.kt]: Probably intellij autoformatting weirdness after the package name change. There's an open issue to use spotless once PRs die down

**Pull request merged**